src/Draco.Compiler/Api/Syntax/ElseClauseSyntax.cs

```diff
@@ -0,0 +1,7 @@
+namespace Draco.Compiler.Api.Syntax;
+public partial class ElseClauseSyntax
```

Comment:
```suggestion
namespace Draco.Compiler.Api.Syntax;

public partial class ElseClauseSyntax
```

Please do this separation everywhere for new files